pgrx/src/datum/date.rs

```diff
@@ -151,8 +151,8 @@ impl Date {
     /// This function will panic, aborting the current transaction, if any part is out-of-bounds.
     pub fn new_unchecked(year: isize, month: u8, day: u8) -> Self {
         let year: i32 = year.try_into().expect("invalid year");
-        let month: i32 = month.try_into().expect("invalid month");
-        let day: i32 = day.try_into().expect("invalid day");
+        let month: i32 = month.into();
+        let day: i32 = day.into();
```

Comment:
These implementations existing are simply incorrect, do not use them.

Reply:
would it make sense to perhaps implement `into()` as `try_into().unwrap()` ?  In that case it would be consistent, and can be replaced.